recipe/meta.yaml

```diff
@@ -38,24 +44,32 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - libtool
-    - pkg-config
-    - automake
-    - autoconf
-    - make
+    - libtool  # [unix]
+    - pkg-config  # [unix]
+    - automake  # [unix]
+    - autoconf  # [unix]
+    - make  # [unix]
     - groff  # [unix]
+    - m2-bash  # [win]
+    - m2-patch  # [win]
+    - m2-gcc-libs  # [win]
   host:
-    - krb5
+    - krb5  # [unix]
     - openssl
-    - libntlm
+    - libdb ==6.2.*  # [win]
+    - libntlm  # [unix]
+    - sqlite  # [win]
+    - ldap3  # [win]
   run:
     - krb5
     - openssl
-    - libntlm
+    - libntlm  # [unix]
+    - libdb  # [win]
     
 test:
   commands:
     - echo "DONE"
+    - if not exist %LIBRARY_LIB%\\libsasl.lib exit 1  # [win]
 
 about:
   home: https://www.cyrusimap.org/sasl/
```

Comment:
maybe you could add yourself to the maintainer list?

Reply:
Thank you for taking a look at this @wolfv, I have added myself to the maintainer list, when you have a moment could you take another look?

Reply:
This slipped my mind, when you have a moment could you take another look @wolfv ?